windtunnel/display.py

```diff
@@ -175,3 +176,50 @@ def show(self):
         Show the visualization.
         """
         self.plt.show()
+
+    @staticmethod
+    def visualize_force_coefficients(file_path: str, coefficient: str):
```

Comment:
move to outside class (it's just a method from this module) -- and maybe call it plot_force_coefficients

Reply:
thanks, I implemented those changes